packages/hash/frontend/src/components/Dropdowns/VersionDropdown.tsx

```diff
@@ -20,10 +19,17 @@ export const VersionDropdown: VoidFunctionComponent<VersionDropdownProps> = ({
   const now = new Date();
 
   return (
-    <select
-      className={styles.AccountSelect}
-      onChange={(event) => onChange(event.target.value)}
-      style={{ width: 200 }}
+    <Box
+      component="select"
+      sx={{
+        padding: "8px 15px",
+        border: "1px solid lightgray",
+        width: 200,
+        borderRadius: "4px",
+      }}
+      onChange={(event: ChangeEvent<HTMLSelectElement>) =>
+        onChange(event.target.value)
+      }
```

Comment:
Yeah I agree. I did this instead because
1) I needed to remove PageSidebar.module.css which this component was dependent on
2) We might eventually rebuild/restyle this component later on.
Will add the `@todo` just in case.

Reply:
Thanks :)